lib/main_containers.pm

```diff
@@ -200,13 +202,13 @@ sub load_container_tests {
         } elsif (get_var('REPO_BCI')) {
             loadtest 'containers/host_configuration';
             loadtest 'containers/bci_repo';
-        }
-        else {
+        } else {
             # Container Host tests
             load_host_tests_podman($run_args) if (/podman/i);
             load_host_tests_docker($run_args) if (/docker/i);
             load_host_tests_containerd_crictl() if (/containerd_crictl/i);
             load_host_tests_containerd_nerdctl() if (/containerd_nerdctl/i);
+            loadtest('containers/kubectl') if ($runtime eq 'kubectl');
```

Comment:
This will work cause we will enable a single job for kubectl. But in this loop, we breakdown `$runtime` into its elements separated by coma. So, in case we decide to run kubectl and helm (for instance) tests together, this test won't be scheduled. So, I would propose:

`loadtest('containers/kubectl') if if (/kubectl/i);`